frontend/src/metabase/query_builder/components/filters/pickers/SelectPicker.styled.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+
+export interface SelectPickerButtonProps {
+  isSelected: boolean;
+}
+
+export const SelectPickerButton = styled.button<SelectPickerButtonProps>`
+  color: ${props => (props.isSelected ? color("white") : color("filter"))};
+  text-align: center;
+  font-weight: 700;
+  width: 100%;
+  height: 95px;
+  border: 1px solid ${color("filter")};
+  border-radius: 0.5rem;
+  background-color: ${props =>
```

Comment:
Do you think grouping properties using `css` would improve readability here, and even more so with multiple properties involving the same prop?

e.g.

#### Separating properties
```ts
  color: ${props => (props.isSelected ? color("white") : color("filter"))};
  background-color: ${props =>
    props.isSelected ? color("filter") : color("white")};
```

#### grouping properties
```ts
  ${({ isSelected }) => {
    if (isSelected) {
      return css`
        color: ${color("white")};
        background-color: ${color("filter")};
      `;
    } else {
      return css`
        color: ${color("filter")};
        background-color: ${color("white")};
      `;
    }
  }};
```

I find grouping properties look more similar to what we've done before https://github.com/metabase/metabase/pull/24058/files#diff-d872445b1460641dff5add8aab8684954ecdb6045ee50e44b6bd39654359c6c1L140-L141.

Reply:
I would personally prefer the first approach since it has less lines of code to read. But I don't mind either.